include/flatbuffers/idl.h

```diff
@@ -1011,6 +1011,8 @@ class Parser : public ParserState {
   IDLOptions opts;
   bool uses_flexbuffers_;
 
+  reflection::AdvancedFeatures advanced_features_;
```

Comment:
enums in C++ are `int` typically, so you'll want to explicitly use a `uint64_t` I'm afraid. You could use `enum class` which allow you to specify the size but those suck for bitfields.

Reply:
I think it's better to use the specific type, even if its more awkward to use. It'd be less awkward if we generated `operator|=` between our bitflags enum and itself...

Regardless, I converted it to the typical style.